lib/membrane_ffmpeg_swresample_plugin/converter.ex

```diff
@@ -122,16 +123,29 @@ defmodule Membrane.FFmpeg.SWResample.Converter do
   end
 
   @impl true
-  def handle_buffer(:input, %Buffer{payload: payload}, _ctx, state) do
+  def handle_buffer(:input, %Buffer{payload: payload, pts: input_pts}, _ctx, state) do
+    input_frame_size = RawAudio.frame_size(state.input_stream_format)
+    output_frame_size = RawAudio.frame_size(state.output_stream_format)
+    ratio = output_frame_size / input_frame_size
+
+    expected_output_frames_count = (byte_size(payload) / input_frame_size * ratio) |> round()
```

Comment:
It is more precise to write `(byte_size(payload) * output_frame_size / (input_frame_size * input_frame_size))` because of limitted accuracy of floats